Jenkinsfile.github

```diff
@@ -146,11 +146,11 @@ pipeline {
                     stage('Publish: Image') {
                         steps {
                             script {
-                                publishCsmDockerImage(image: "cray-${env.NAME}", tag: "${image_version}-p${PYTHON_VERSION}", isStable: isStable)
+                                publishCsmDockerImage(image: "${env.NAME}", tag: "${image_version}-p${PYTHON_VERSION}", isStable: isStable)
 
                                 // Only publish the simple version images on the latest/newest base image.
                                 if ("${PYTHON_VERSION}" == "${mainPythonVersion}") {
```

Comment:
Since the image is not installing the RPM, both the `Build: Image` and `Publish: Image` step should move _out_ of the `matrix`. Then this conditional can go away entirely.